skills/eval-analyze/scripts/discover.py

```
@@ -0,0 +1,139 @@
+    fm = yaml.safe_load(parts[1]) or {}
+    skill_name = fm.get("skill", "")
+    stored_hash = fm.get("skill_hash", "")
+
+    if not skill_name or not stored_hash:
+        print("STALE: missing skill or hash")
+        sys.exit(1)
+
+    for skills_dir in [Path(".claude/skills"), Path("skills")]:
+        skill_path = skills_dir / skill_name / "SKILL.md"
+        if skill_path.exists():
+            current_hash = hashlib.md5(skill_path.read_bytes()).hexdigest()[:12]
```

Ruff (0.15.7)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)